testing/integration/container_cmd_test.go

```diff
@@ -121,7 +121,7 @@ func TestContainerCMD(t *testing.T) {
 		healthy, err = agentFixture.IsHealthy(ctx)
 		return healthy
 	},
-		5*time.Minute, time.Second,
+		10*time.Minute, time.Second,
```

Comment:
This is a crazy long timeout, are we sure it needs to wait this long?

If the agent takes 10 minutes to become healthy it is almost certainly a bug.

The agent should become healthy in here seconds.